aries-site/src/pages/templates/forms/managing-child-objects.mdx

```diff
@@ -0,0 +1,74 @@
+and lead to quite lengthy forms.
+
+This pattern improves productivity by focusing a user's attention solely on the current object 
+of interest, and uses collapsible objects to show or hide detail as needed. The 
+intent is that collapsible objects reduce noise, clutter, and the resulting cognitive overload 
+by hiding extraneous detail &mdash; and an easy means to view detail when needed.
+
+## Anatomy
+
+The following illustrates the elements composing a child object in its collapsed and expanded 
+states. View the [use case examples](#use-case-examples) to see how child objects behave and are presented in a form 
+context.
+
+### Collapsed state
+
+[Diagram to be inserted]
+
+| Label | Region         | Purpose                                                    | Required? | Notes |
+| ----- | -------------- | ---------------------------------------------------------- | --------- | ----- |
+|       | Container      | Identfies the child and controls the show detail behavior. | Required  | The entirety of the container is focusable and shows detail with a mouse click, Enter key, or Spacebar. The aria-label should read "Show and edit detail." |
+|       | Label          | Uniquely identifies each child object.                     | Required  | Should be implemented as a semantically correct heading to allow screen reader users to traverse between children. |
+|       | Expand icon    | Visual treatment indicating show detail behavior.          | Required  | Icon name is "FormDown". | 
+|       | Values summary | A preview summarizing the object's detail.                 | Optional  |       |
+|       | Border         | Provides visual distinction from other children.           | Required  |       |
+
+### Expanded state
+
+[Diagram to be inserted]
+
+| Label | Region        | Purpose                                                     | Required? | Notes |
+| ----- | ------------- | ----------------------------------------------------------- | --------- | ----- |
+|       | Header        | Identifies the child and controls the hide detail behavior. | Required  | The entirety of the container is focusable and hides detail with a mouse click, Enter key, or Spacebar. The aria-label should read "Hide detail." |
+|       | Label         | Uniquely identifies each child object.                      | Required  | Should be implemented as a semantically correct heading to allow screen reader users to traverse between children. |
+|       | Collapse icon | Visual treatment indicating hide detail behavior.           | Required  | Icon name is "FormUp". |
+|       | Body          | Flexible region for presenting an object's detail.          | Required  | Detail presentation may vary by the use case. |
+|       | Remove button | Removes/deletes the child from its parent.                  | Required  |       | 
+|       | Border        | Provides visual distinction from other children.            | Required  |       |
+
+## Use case examples
```

Comment:
Small alignment thing -- in cards and layer I've just been calling this section "Examples". Thinking we should align across pages. Do you have a strong preference in one direction or the other? Or do you view Examples vs Use case examples as distinct things?

Reply:
🤔  -- I noticed that when reviewing the layer work as well. 

I'm not sure which way to think about it. "Examples" provides more flexibility as it would be inclusive of "example use cases" but allow for non-use case oriented executions (having a hard time of thinking what those might be at the moment).

Perhaps we simplify with "Examples" for the time being?

Reply:
I'm good with that.

Reply:
Done.